packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/ResetPasswordModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
```

Comment:
It's not wrong. But I think It's good to think to use React-hook-form for all form for the next conception. So I approve but If you want refactor I am here.

Reply:
Ah ok you have not field , sorry 😅